easybuild/easyconfigs/l/libdap/libdap-3.20.11-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'libdap'
+version = '3.20.11'
+
+homepage = 'https://www.opendap.org/software/libdap'
+description = """A C++ SDK which contains an implementation of DAP 2.0 and
+ DAP4.0. This includes both Client- and Server-side support classes."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://www.opendap.org/pub/source/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['850debf6ee6991350bf31051308093bee35ddd2121e4002be7e130a319de1415']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('Bison', '3.8.2'),
+    ('flex', '2.6.4'),
+]
+dependencies = [
+    ('cURL', '8.0.1'),
+    ('libxml2', '2.11.4'),
+    ('libtirpc', '1.3.3'),
+    ('PCRE', '8.45'),
+    ('util-linux', '2.39'),
+]
+
+configopts = 'TIRPC_LIBS="-ltirpc"'
+
+
+sanity_check_paths = {
+    'files': ['bin/getdap', 'bin/getdap4', 'bin/dap-config', 'lib/%(name)s.a', 'lib/%(name)s.so'],
```

Comment:
Use 'lib/lib%%(name)s.%s' % SHLIB_EXT  instead of hardcoding ".so"

Reply:
Updated the source branch.
Is there an equivalent variable for ".a" extensions?

Reply:
No we do not have anything for .a.